src/Integrations/class-facebook-instant-articles.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Facebook Instant Articles integration class
+ *
+ * @package Parsely\Integrations
+ * @since 2.6.0
+ */
+
+namespace Parsely\Integrations;
+
+/**
+ * Integrates Parse.ly tracking with the Facebook Instant Articles plugin.
+ *
+ * @since 2.6.0 Moved from Parsely class to this file.
+ */
+final class Facebook_Instant_Articles implements Integration {
+	const REGISTRY_IDENTIFIER   = 'parsely-analytics-for-wordpress';
+	const REGISTRY_DISPLAY_NAME = 'Parse.ly Analytics';
+
+	/**
+	 * Apply the hooks that integrate the plugin or theme with the Parse.ly plugin.
+	 *
+	 * @since 2.6.0
+	 */
+	public function integrate() {
+		if ( defined( 'IA_PLUGIN_VERSION' ) ) {
+			add_action( 'instant_articles_compat_registry_analytics', array( $this, 'insert_parsely_tracking' ) );
+		}
+	}
+
+	/**
+	 * Add Parse.ly tracking to Facebook instant articles.
+	 *
+	 * @since 2.6.0
+	 *
+	 * @param array $registry The registry info for fbia.
+	 * @return string
+	 */
+	public function insert_parsely_tracking( &$registry ) {
+		$options = get_option( \Parsely::OPTIONS_KEY );
+		if ( ! ( $options['apikey'] ) ) {
+			return;
+		}
+
+		$registry[ self::REGISTRY_IDENTIFIER ] = array(
+			'name'    => self::REGISTRY_DISPLAY_NAME,
+			'payload' => $this->get_embed_code( $options['apikey'] ),
+		);
+	}
+
+	/**
+	 * Get the payload / embed code.
+	 *
+	 * @since 2.6.0
+	 *
+	 * @param string $api_key API key.
+	 *
+	 * @return string
+	 */
+	public function get_embed_code( $api_key ) {
+		$embed_code = '<script>
```

Comment:
Could we return `$embed_code` directly without having to declare the variable?

Reply:
Yes, done :-)